crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/util/CrimeApplicationUtil.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.laa.crime.applications.adaptor.util;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import lombok.experimental.UtilityClass;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+@UtilityClass
+public class CrimeApplicationUtil {
```

Comment:
Maybe rename this class to `CrimeApplicationHttpUtil` to make it's responsibility clearer and more specific.